internal/pally/counter.go

```diff
@@ -23,7 +23,9 @@ package pally
 import (
 	"fmt"
 
+	"github.com/golang/protobuf/proto"
```

Comment:
We don't get to choose the proto library directly, since it's a type passed to us by prometheus. Since we're locked into using the Google proto library, I'd prefer to also use their helpers.

This whole code base is moving out of YARPC, so I don't think it's super-critical to keep it consistent. Does that sound okay to you?